docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2414,6 +2416,33 @@ Here is an example:
 
   In the final version of the observers, following the noise functions, please have all data as strings. Age must be rounded down to a whole number before applying noise.
 
+**Generating WIC IDs**
+
+To generate the WIC family ID for simulated households observed in WIC,
+sort them by when the household was first observed in WIC (it does not matter how ties are broken),
+then assign them consecutive integers.
+Zero-pad these integers to 9 digits before noising.
+
+.. note::
+  The `actual process for generating WIC family IDs <https://web.archive.org/web/20230601182852/https://www.wichands.com/Content/Docs/Help/ClinicUserManual/Module1/index.html?page=family-id-number.html>`_  involves using the year in combination with
+  a consecutive integer.
+  Our approach is simpler, while still capturing that these
+  numbers are likely to be similar and easy to mistake for one another.
+  In fact, we will tend to overstate how much this is the case by not changing
+  the prefix every year.
```

Comment:
Is there a reason we're not including the year? It seems straightforward to do so.

Reply:
Eh, I think it's not worth it. It's a bit more work, will be weird because we won't have anyone with an (implied) year before our simulation start, and doesn't add much.